uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/CamelEditorDataFormatTest.java

```diff
@@ -96,6 +98,29 @@ public CamelEditorDataFormatTest(String dataFormat) {
 		this.dataFormat = dataFormat;
 	}
 
+	/**
+	 * Checks if used Camel Version is newer than input.
+	 *
+	 * @param version
+	 *                    Integer representing required version for true return. f.e. 7 -> 7.0.0, 73 -> 7.3.0, 731 ->
+	 *                    7.3.1
+	 * @return true if actual version is equal to required or newer
+	 * @return false if actual version is older then required
+	 */
+	private static boolean IsCamelVersionNewerThan(Integer version) {
```

Comment:
I'm wondering if org.osgi.framework.Version is not available and in this case, there might be a shorter solution. I think it worth checking

Reply:
@apupier Thanks for tip, I will look at this.